example3/emperror.dev/errors/match/BUILD

```diff
@@ -9,5 +9,6 @@ go_library(
     name = "match",
     srcs = [":source"],
     visibility = ["PUBLIC"],
-    deps = ["//example3/emperror.dev:errors"],
+    deps = ["//example3/emperror.dev/errors"],
+    _import_path = "emperror.dev/errors/match",
```

Comment:
Was thinking if I should make this public

Reply:
Third-party go packages definitely need it.